ack/readers/google_dbm/reader.py

```diff
@@ -55,9 +55,12 @@ def __init__(self, access_token, refresh_token, client_secret, client_id, **kwar
 
         self.kwargs = kwargs
 
-        check_date_range_definition_conformity(
-            self.kwargs.get("start_date"), self.kwargs.get("end_date"), self.kwargs.get("day_range")
-        )
+        if self.kwargs.get("query_frequency") == "ONE_TIME":
+            check_date_range_definition_conformity(
+                self.kwargs.get("start_date"), self.kwargs.get("end_date"), self.kwargs.get("day_range")
+            )
+        else:
+            check_date_range_definition_conformity(self.kwargs.get("start_date"), self.kwargs.get("end_date"), None)
```

Comment:
Good job ! 👍 

Quick question : why do we need these two cases here ?

Also, for my understanding : does the start_date and end_date parameters represent the same thing in custom_reports and custom_scheduled_reports ? Or does it represent the period of the queried data in one case, and the period on which the scheduled report should run in the second case ?

Reply:
Indeed, start date and end date represent the period of the queried data in one case, and the period on which the scheduled report should run in the second case. It might be better to add two arguments (like scheduled-end-date + scheduled-start-date). 
What do you think ? 

I needed these two cases because when querying a scheduled report, we should be able to have both a day-range (Last_7_Days for example) and a couple (start-date, end-date).

Reply:
Yes, it might be clearer to have dedicated date arguments for the custom_scheduled_report 😊 Otherwise it can be quite confusing.

Got it for the 2 cases! Then shouldn't we add a check to verify that the user did not provide a day_range if the request type is custom_scheduled_report (because it does not make sense)?

Reply:
In fact what doesn't make sense is having both a day_range and a couple(start_date, end_date) for "normal" reports.

But for scheduled reports, we need to have both a day_range and (schedule_start_date, schedule_end_date): 
query on "LAST_7_DAYS" (day-range) between 07/07/21 (schedule-start-date) and 07/08/21 (schedule-end-date) on a "DAILY" basis (=frequency) for example. Thus I need to create a specific check for scheduled reports (different from check_date_range_definition_conformity)